test/suite/test_tiered07.py

```diff
@@ -147,7 +160,11 @@ def test_tiered(self):
 
         # Create new table with new name.
         self.pr('create new table')
-        self.session.create(self.newuri, 'key_format=S')
+        self.session.create(self.newuri, 'key_format=S,value_format=S,tiered_storage=(name=none)')
```

Comment:
Maybe we should create another table and leave `self.newuri` without any changes in config?